pkg/apiserver/certificate/common.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certificate
+
+import "time"
+
+const (
+	CAConfigMapName    = "antrea-ca"
+	DefaultServiceName = "antrea"
+)
+
+type CAConfig struct {
+	// Name of the ConfigMap that will hold the CA certificate that signs the TLS
+	// certificate of antrea-controller.
+	CAConfigMapName string
+
+	// APIServiceNames contains all the APIServices backed by antrea-controller.
+	APIServiceNames []string
+
+	// ValidatingWebhooks contains all the ValidatingWebhookConfigurations backed by antrea-controller.
+	ValidatingWebhooks []string
+
+	// MutationWebhooks contains all the MutationWebhooks backed by antrea-controller.
+	MutationWebhooks []string
+
+	// OptionalMutationWebhooks contains all the OptionalMutationWebhooks backed by antrea-controller.
+	OptionalMutationWebhooks []string
+
+	// CrdsWithConversionWebhooks contains all the ConversionWebhooks backed by antrea-controller.
+	CrdsWithConversionWebhooks []string
```

Comment:
Should we use CRDsWithConversionWebhooks if we want to stick to all upper cases for abbrs.?

Reply:
sure, updated it to `CRDsWithConversionWebhooks`